enterprise/frontend/src/embedding-sdk/test/CommonSdkCorsStoryWrapper.tsx

```diff
@@ -0,0 +1,17 @@
+import type { Story } from "@storybook/react";
+
+import { MetabaseProvider, type SDKConfig } from "embedding-sdk";
+
+const METABASE_INSTANCE_URL =
+  (window as any).METABASE_INSTANCE_URL || "http://localhost:3000";
+
+const DEFAULT_CONFIG: SDKConfig = {
+  metabaseInstanceUrl: METABASE_INSTANCE_URL,
+  jwtProviderUri: `${METABASE_INSTANCE_URL}/sso/metabase`,
+};
```

Comment:
duplicated from `CommonSdkStoryWrapper.tsx`, but removed the `fetchRequestToken`, so this wrapper doesn't sign JWT and let Cypress do that itself.